### PR TITLE
disables carrier

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
This is just Chazs PR, which he closed for his own personal reasons but I also fixed defiler check

## Why It's Good For The Game
carrier is literally broken as heck, it should die

## Changelog
:cl:
del: Removed carrier
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -189. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by BeloneX** [comment]
Atmoize this PR.

**Comment by Pariah919** [comment]
>Atmoize this PR.
>
what even is there to atomize

**Comment by BeloneX** [comment]
> > Atmoize this PR.
> 
> what even is there to atomize

The defiler bug fix and the Carrier config.

**Comment by Pariah919** [comment]
>The defiler bug fix and the Carrier config.
>
if you actually bothered to read the pr and click on files you'd realize the changes are relevant to the PR (disabling carrier), otherwise please go read it. Thank you.

File: code/_globalvars/lists/mobs.dm [hunk 1]
```diff
@@ -46,8 +46,7 @@ GLOBAL_LIST_EMPTY(all_species)
 
 GLOBAL_LIST_EMPTY_TYPED(xeno_caste_datums, /list/datum/xeno_caste)
 GLOBAL_LIST_INIT(xeno_types_tier_one, list(/mob/living/carbon/xenomorph/runner, /mob/living/carbon/xenomorph/drone, /mob/living/carbon/xenomorph/sentinel, /mob/living/carbon/xenomorph/defender))
-GLOBAL_LIST_INIT(xeno_types_tier_two, list(/mob/living/carbon/xenomorph/hunter, /mob/living/carbon/xenomorph/warrior, /mob/living/carbon/xenomorph/spitter, /mob/living/carbon/xenomorph/hivelord, /mob/living/carbon/xenomorph/carrier, /mob/living/carbon/xenomorph/bull))
-GLOBAL_LIST_INIT(xeno_types_tier_three, list(/mob/living/carbon/xenomorph/ravager, /mob/living/carbon/xenomorph/praetorian, /mob/living/carbon/xenomorph/boiler, /mob/living/carbon/xenomorph/Defiler, /mob/living/carbon/xenomorph/crusher, /mob/living/carbon/xenomorph/shrike))
```
Comment (BraveMole) [inline code/_globalvars/lists/mobs.dm:50]: this removes all T3, kek

**Comment by Naaanii** [comment]
Salt PR?

**Comment by TiviPlus** [comment]
funny

**Pull request closed without merging**